docs/static/settings-file.asciidoc

```diff
@@ -63,7 +63,17 @@ modules:
 
 IMPORTANT: If the <<command-line-flags,command-line flag>> `--modules` is used, any modules defined in the `logstash.yml` file will be ignored.
 
-The `logstash.yml` file includes the following settings. 
+[discrete]
+[[logstash-yml-settings]]
+=== Logstash settings
+
+The `logstash.yml` file includes these settings. 
+
+[discrete]
+[[settings-pq]]
+==== Persistent queue settings
+
+include::./settings/dlq-settings.asciidoc[]
```

Comment:
`ERROR: static/settings-file.asciidoc: line 76: include file not found: /tmp/docsbuild/8gG0huEE8e/logstash/docs/static/settings/dlq-settings.asciidoc`